src/features/routing/model.ts

```diff
@@ -51,6 +51,33 @@ export abstract class SRoutableElement extends SChildElement {
             height: 0
         }), EMPTY_BOUNDS);
     }
+
+    /**
+     * Determine whether this routable element is visible in the current canvas.
+     */
+    isVisible(route: Point[] = this.routingPoints, context?: RenderingContext): boolean {
```

Comment:
Why is this a method while `isVisible` for other elements is a function?
And why is it in the SModel? This introduces a dependency from model to view (RenderingContext).

Reply:
This check depends explicitly on `SRoutableElement`, which is a class, while the general `isVisible` depends on `BoundsAware`, which is an interface.

This model is in `features`, I think the dependency to an interface from `base/views/view` shouldn't hurt.